### ISSUE TITLE
[Design suggestion] Day start should be morning

### ISSUE THREAD

**Issue by Raton-Laveur** [Gabboxl/FluentWeather#87]
When clicking on a daily forecast, the hourly forecasts below it start at midnight. 

Suggestion : should start on morning time (such as 8AM), or provide an option to choose day start time. 

Current view starting at midnight isn't useful, as users generally want to know how the selected day will start. It also forces users with cramped views/vertical windows or displays to scroll as a first action after selecting a daily forecast to see useful hourly forecast that isn't midnight-to-morning.

**Comment by Gabboxl** [comment]
You're right, but there are also people who wake up very early... So it must be an option if implemented. I will see
